src/markdown/tutorial/part-2/11-ember-data.md

```diff
@@ -157,6 +157,29 @@ It is worth pointing out that EmberData provides a `store` *[service](../../../s
 export { default } from 'ember-data/store';
 ```
 
+```run:file:patch hidden=true lang=js cwd=super-rentals filename=app/app.js
```

Comment:
Hidden patches?! Ooooooo

Reply:
You'll find a bunch of them -- mostly doing setup for the tutorial infrastructure